include/librealsense2/dds/topics/notifications/notifications-topic-files.cmake

```diff
@@ -0,0 +1,10 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+set(NOTIFICATION_TOPIC_FILES
```

Comment:
I think we should match, filename to define -- so it should be `NOTIFICATIONS_TOPIC_FILES`?

Reply:
👍

Reply:
Done